tests/unit/pomodoro_test.sh

```diff
@@ -380,4 +387,54 @@ function test_show_tags()
   assert_equals_helper 'Wrong output' "$LINENO" "$expected" "$output"
 }
 
+function test_fetch_last_pomodoro_session()
+{
+  local output
+  local expected
+  local values
+  local columns='("date","time","duration","tag_name","description")'
+
+  values="('2024-05-27','17:36:20','120','fake_tag','this is not a drill')"
+  sqlite3 "${KW_DATA_DIR}/kw.db" -batch "INSERT INTO 'pomodoro_report' ${columns} VALUES ${values} ;"
+  output=$(fetch_last_pomodoro_session)
+  expected=(
+    'Last pomodoro session:'
+    '- Duration: 120s'
+    '- Tag: fake_tag'
+    '- Description: this is not a drill'
+  )
+  compare_command_sequence 'Last session' "$LINENO" 'expected' "$output"
+
+  values="('2023-05-27','14:36:20','120','true_tag','this is a drill')"
+  sqlite3 "${KW_DATA_DIR}/kw.db" -batch "INSERT INTO 'pomodoro_report' ${columns} VALUES ${values} ;"
+  output=$(fetch_last_pomodoro_session)
+  expected=(
+    'Last pomodoro session:'
+    '- Duration: 120s'
+    '- Tag: fake_tag'
+    '- Description: this is not a drill'
+  )
+  compare_command_sequence 'Older session' "$LINENO" 'expected' "$output"
```

Comment:
Let's add a little bit of entropy between these tests, create some new entries between the sessions, and ensure that the `--repeat` will get the new session with different parameters (time, tag, and description).